src/usr/initservice/istepdispatcher/HBconfig

```diff
@@ -13,4 +13,7 @@ config SIO_ISTEP_CONTROL
     help
         Allows istep control via SIO scratch registers.
         Typical usage is via mailbox (scom) scratch registers.
-
+config PORT80_DEBUG
```

Comment:
I think we should have a more descriptive tag, how about  ISTEP_LPC_PORT80_DEBUG?